ansible/services.yml

```diff
@@ -70,7 +70,7 @@
     nginx_sites:
       vagrant:
          - listen 80
-         - server_name *.dev
+         - "server_name *.{{ hostname }}"
```

Comment:
why?

Reply:
I mean, this is stupid because I can't change the hostname without re-provisioning. What's your reasons behind the change?

Reply:
don't want to listen to * because now you can put random tld on your hostname. do you want to extract tld from it only?

Reply:
What about `server_name  _;` ?

`*{tld}` does not looks working to me, "vagrant" is also valid host, but has not tld.

Reply:
then it will go to `server_name *;` as tld will be ""

Reply:
you can not also set _ as vagrant will refuse to create a vm

Reply:
with what reason ?

Reply:
* The hostname set for the VM should only contain letters, numbers,
hyphens or dots. It cannot start with a hyphen or dot.

Reply:
Ehm, I mean to use the special nginx server_name `_` this is catch all.
You'd probably also need to change the site name from `vagrant` to  `default` because of https://github.com/jdauphant/ansible-role-nginx/blob/master/defaults/main.yml#L43

I'm talking about nginx here, not vagrant hostname.

Reply:
yes but do we need catch all?